test/functional/validation-functions-and-decorators.spec.ts

```diff
@@ -1264,7 +1264,7 @@ describe('MinDate function constraint', () => {
 
   it('should return error object with proper data', () => {
     const validationType = 'minDate';
-    const message = 'minimal allowed date for someProperty is ' + constraintToString(constraint);
+    const message = 'minimal allowed date for someProperty is ' + constraintToString(constraint());
```

Comment:
Does this work is the value is a Date? It seems this is made to work with `() => Date`

Reply:
Yes, it is working for both cases. When I run `npm test` against PR, it sucessfully ends.

Reply:
We should invoke constraint because it is a function in line 1240. If constraint is not function (it is Date), other test case line 1173 is guarantees that it works as expected.